src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -37,6 +38,26 @@ namespace Microsoft.PowerFx.Core.Functions
     [ThreadSafeImmutable]
     internal abstract class TexlFunction : IFunction
     {
+        internal virtual BlankHandlerPolicy GetArgBlankHandlerPolicy(int index)
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

nit: since the class is already internal, this can just be public.

Reply:
updated.